controllers/dashboard_test.go

```diff
@@ -30,17 +33,153 @@ var _ = Describe("DashboardController", func() {
 	})
 
 	AfterEach(func() {
+		TruncateTable("keyword")
 		TruncateTable("user")
 	})
 
 	Describe("GET /dashboard", func() {
 		Context("when the user has already signed in", func() {
 			It("renders with status 200", func() {
-				user := FabricateUser("dev@nimblehq.co", "password")
+				user := FabricateUser(faker.Email(), faker.Password())
 				response := MakeAuthenticatedRequest("GET", "/dashboard", nil, user)
 
 				Expect(response.StatusCode).To(Equal(http.StatusOK))
 			})
+
+			Context("given 12 records of the keyword in the database", func() {
+				It("shows 2 pages in pagination", func() {
```

Comment:
Since it's not a UI test, I would refrain from using the wording `show` 😅 How about:

```suggestion
				It("lists 2 pages in pagination", func() {
```

The same applies to all tests. It's a minor change though.